### PR TITLE
Cleanup readme

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- Cleanup readme from unused make targets, add overview section and update description how to run or test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,68 +5,48 @@ K8s image authenticity validator
...
-1. Install the CRDs into the cluster:
+### Run locally
+Install helm charts via:
 
 ```sh
 make install
 ```
 
-2. Run your controller (this will run in the foreground, so switch to a new terminal if you want to leave it running):
+Install helm charts with locally build images:
```
Comment (dbadura) [inline README.md:39]: I would add note `works only with k3d`

File: README.md [hunk 2]
```diff
@@ -79,6 +59,22 @@ make manifests
 
 More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
 
+### Test strategy
```
Comment (dbadura) [inline README.md:62]: Could you fix line:
**NOTE:** Run `make --help` for more information on all potential `make` targets
to:
**NOTE:** Run `make help` for more information on all potential `make` targets

File: README.md [hunk 3]
```diff
@@ -79,6 +59,22 @@ make manifests
...
+Run unit tests
+```sh
+make verify
+```
+Start k3d instance locally and run integration  tests
+```sh
+make k3d-integration-test
+```
+
+Run integration tests on your current kubeconfig context
```
Comment (dbadura) [inline README.md:73]: This target request setup 2 envs, so it's not good candidate to run by user.

File: README.md [hunk 4]
```diff
@@ -79,6 +59,22 @@ make manifests
...
+```sh
+make verify-on-cluster
+```
```
Comment (dbadura) [inline README.md:76]:
```suggestion
Start k3d instance locally and run integration  tests
```sh
make k3d-integration-test
```
or if you have k3d with warden
Run integration tests on your current kubeconfig context
```sh
make run-integration-tests
```
```

NOTE: something is broken with suggestion!

**Review by dbadura — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -5,68 +5,48 @@ K8s image authenticity validator
...
-1. Install the CRDs into the cluster:
+### Run locally
+Install helm charts via:
 
 ```sh
 make install
 ```
 
-2. Run your controller (this will run in the foreground, so switch to a new terminal if you want to leave it running):
+Install helm charts on local k3d instance with locally built images:
```
Comment (dbadura) [inline README.md:39]:
```suggestion
Install helm charts on k3d instance with locally built images:
```

**Review by dbadura — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -75,10 +55,26 @@ If you are editing the API definitions, generate the manifests such as CRs or CR
...
 ```
 
-**NOTE:** Run `make --help` for more information on all potential `make` targets
+**NOTE:** Run `make help` for more information on all potential `make` targets
 
 More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
 
+### Test strategy
+
+Run unit tests
```
Comment (grego952) [inline README.md:64]: If the tests should run one after, then change the steps to the ordered list. If not, then leave it as it is.
```suggestion
1. Run unit tests
```

File: README.md [hunk 7]
```diff
@@ -75,10 +55,26 @@ If you are editing the API definitions, generate the manifests such as CRs or CR
...
+```sh
+make verify
+```
+Start k3d instance locally and run integration tests
```
Comment (grego952) [inline README.md:68]:
```suggestion
2. Start the k3d instance locally and run integration tests
```

File: README.md [hunk 8]
```diff
@@ -75,10 +55,26 @@ If you are editing the API definitions, generate the manifests such as CRs or CR
...
+```sh
+make k3d-integration-test
+```
+
+If you have k8s instance with warden installed already run integration tests with:
```
Comment (grego952) [inline README.md:73]:
```suggestion
If you have the k8s instance with Warden installed already, run integration tests with the following command:
```

File: README.md [hunk 9]
```diff
@@ -75,10 +55,26 @@ If you are editing the API definitions, generate the manifests such as CRs or CR
...
 ```
 
-**NOTE:** Run `make --help` for more information on all potential `make` targets
+**NOTE:** Run `make help` for more information on all potential `make` targets.
 
 More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
 
+### Test strategy
+
+1. Run unit tests
```
Comment (grego952) [inline README.md:64]:
```suggestion
Run unit tests
```

File: README.md [hunk 10]
```diff
@@ -75,10 +55,26 @@ If you are editing the API definitions, generate the manifests such as CRs or CR
...
+```sh
+make verify
+```
+2. Start the k3d instance locally and run integration tests
```
Comment (grego952) [inline README.md:68]:
```suggestion
Start the k3d instance locally and run integration tests
```

**Pull request merged**